hummingbot/hummingbot
# [ADD] Polkadex Exchange Connector

**Before submitting this PR, please make sure**:


**A description of the changes proposed in the pull request**:
This PR adds `Polkadex Connector` into the hummingbot, There are two dependencies that are needed to use the `Polkadex Connector`: 
1. `gql` - GraphQL client for python, which we are using to perform queries. 
2. `substrate-interface` - Python library for interfacing with a `Substrate` Node or in this case `Polkadex` Node. 

**Tests performed by the developer**:
We have tested the connector against `Polkadex Orderbook` in our current `Kaizen` testnet and performed the following
1. Place Orders
2. Cancel Orders 
3. Run `Pure Market Making` strategy 
4. Fill `Hummingbot` Orders
5. Place a better bid price than hummingbot and vice versa 

In all of these scenarios, Hummingbot had expected behavior. 


**Tips for QA testing**:
Before performing QA testing, you would have to go through how to use `Polkadex Orderbook` to easily get started with using hummingbot.
